techmatters/flex-plugins
# CHI-2223: Unsaved confirm dialog on close POC

# Merge #1745  before review

## Description

Whilst it might not capture all the edge cases, this demonstrates a simple & robust 


### Related Issues
CHI-2223

### Verification steps

Edit various aspects of cases / contacts and close the browser window, verify the standard browser confirm dialog appears when edits are unsaved
